### PR TITLE
Fixed localization issue in <database> --> Operations window

### PR DESCRIPTION
**Pull request by Anshu-xLR**
As pointed out in [Bug #4816](https://sourceforge.net/p/phpmyadmin/bugs/4816/), fixed a tiny localization issue in <database> --> Operations window

Signed-off-by: Anshuman Verma anshuman@shuffleconnect.com

### CODE REVIEW

**Comment by ibennetch** [comment]
Thanks for resubmitting as a pull request! Someone will review and comment or merge when they have an opportunity.

File: libraries/operations.lib.php [hunk 1]
```diff
@@ -237,7 +237,7 @@ function PMA_getHtmlForChangeDatabaseCharset($db, $table)
         $html_output .= PMA_Util::getImage('s_asci.png') . '&nbsp';
     }
     $html_output .= '<label for="select_db_collation">' . __('Collation')
-        . ':</label>' . "\n"
```
Comment (ibennetch) [inline libraries/operations.lib.php]: Shouldn't this also move the colon to the translatable string?

Comment (lem9) [inline libraries/operations.lib.php]: We don't really need the colon; in fact, in the RFE, the user believed it would be more logical to remove the colon.

Comment (ibennetch) [inline libraries/operations.lib.php]: Then should we also remove the colons in some of the other labels? :)

Comment (lem9) [inline libraries/operations.lib.php]: Indeed; do you want me to work on this?

Comment (ibennetch) [inline libraries/operations.lib.php]: I won't have an opportunity until this evening so sure, go right ahead.

Comment (lem9) [inline libraries/operations.lib.php]: Done.

**Pull request merged**